TRANSFORMERS/VECTOR_MANIPULATION/VECTOR_INDEXING/VECTOR_INDEXING.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+from flojoy import flojoy, Vector, Scalar
+
+
+@flojoy
+def VECTOR_INDEXING(
+    default: Vector,
+    index: int = 0,
+) -> Scalar:
+    """The VECTOR_INDEXING node returns the value of the Vector at the
+    requested index.
+
+    Inputs
+    ------
+    v : Vector
+        The input vector to index.
+
+    Parameters
+    ----------
+    index : int
+        The index of the vector to return.
+
+    Returns
+    -------
+    Scalar
+        The scalar index of the input vector.
+    """
+    assert (
+        len(default.v) >= index
+    ), "The index parameter must be less than or equal to the length of the Vector."
+    assert index >= 0, "The index parameter greater than zero."
```

Comment:
Could you please attach a screenshot of what kinda error message will be displayed on the flowchart if they get triggered?

Reply:
![image](https://github.com/flojoy-ai/nodes/assets/41165864/36410f26-a0bf-4bcb-b5d7-b720047d7a9e)

![image](https://github.com/flojoy-ai/nodes/assets/41165864/7262437b-fc31-4488-abaa-93f44fc863c0)